src/deadline/maya_adaptor/MayaClient/maya_client.py

```diff
@@ -10,11 +10,20 @@
 # so that importing just the adaptor_runtime_client should work.
 try:
     from adaptor_runtime_client import HTTPClientInterface  # type: ignore[import]
+except (ImportError, ModuleNotFoundError):
+    try:
+        from openjd.adaptor_runtime_client import HTTPClientInterface  # type: ignore[import]
```

Comment:
Is there a reason for keeping HTTPClientInterface specifically instead of just doing `ClientInterface` always and changing all references. That should just automatically use HTTPClientInterface if on Linux

Reply:
Definitely want to clean it up, can do it in a follow-up